helm-system-packages-brew.el

```diff
@@ -0,0 +1,214 @@
+;;; helm-system-packages-brew.el --- Helm UI for Mac OS ' homebrew. -*- lexical-binding: t -*-
+
+;; Copyright (C) 2012 ~ 2014 Thierry Volpiatto <thierry.volpiatto@gmail.com>
+;;               2017 ~ 2018 Pierre Neidhardt <ambrevar@gmail.com>
+
+;; Author: Arnaud Hoffmann <tuedachu@gmail.com>
+;; URL: https://github.com/emacs-helm/helm-system-packages
+;; Version: 1.8.0
+;; Package-Requires: ((emacs "24.4") (helm "2.8.6"))
+;; Keywords: helm, packages
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Helm UI for Mac OS' homebrew.
```

Comment:
"macOS".

Reply:
Should be addressed by https://github.com/tuedachu/helm-system-packages/pull/1.